### ISSUE TITLE
WCAG 2.2 AA audit

### ISSUE THREAD

**Issue by brandonhundt** [PRX/feeder.prx.org#1418]
Prompted by a discussion with a potential client that needs to be WCAG 2.2 AA compliant. Regardless of if they choose Dovetail or not, we should ensure our pages in Feeder are compliant as that aligns with latest government standards, as we have a lot of stations associated with universities.

*Lighthouse is not necessarily built to this standard, test should be completed with the [Deque Axe web extension](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
